src/disposable-modal.js

```diff
@@ -3,19 +3,26 @@ import ReactDOM from "react-dom";
 import Cancelable from "react-disposable-decorator";
 import mountComponent from "disposable-component";
 
+@Cancelable
 class _Modal extends React.Component {
   componentDidMount() {
-    const children = this.props.children;
-
-    this.props.cancelWhenUnmounted(
-      createCancelableModal(function Portal() {
-        return children;
-      }, this.props).subscribe(() => {})
-    );
+    if(!ReactDOM.createPortal) {
```

Comment:
I like this better than looking directly at the version prop

Reply:
Yeah, agreed